Sources/Algorithms/Chain.swift

```diff
@@ -270,45 +270,57 @@ extension Chain: BidirectionalCollection
   }
 }
 
-extension Chain: RandomAccessCollection
+extension Chain2: RandomAccessCollection
   where Base1: RandomAccessCollection, Base2: RandomAccessCollection {}
-extension Chain: LazySequenceProtocol where Base1: LazySequenceProtocol {}
+extension Chain2: LazySequenceProtocol where Base1: LazySequenceProtocol {}
```

Comment:
I believe this conformance should be removed altogether with `chain` being a free function.

Reply:
Yes! Another wrinkle in the various keep-it-lazy shenanigans.